vars/runPipeline.groovy

```diff
@@ -462,6 +466,7 @@ def call() { timeout(time: 1, unit: 'HOURS') {
         "libmir/mir-algorithm", // 17s
         "CyberShadow/ae", // 22s
         "dlang-community/DCD", // 23s
+        "BBasile/iz",
```

Comment:
Could you please set this to `12s` and put it at the according position?

Or enable PR push to your PRs ;-)

```
 ! [remote rejected] HEAD -> add-iz (permission denied)
error: failed to push some refs to 'git@github.com:BBasile/ci'
```